pkgs/servers/pulseaudio/qpaeq.nix

```diff
@@ -0,0 +1,51 @@
+{ mkDerivation
+, makeDesktopItem
+, python3
+, fetchurl
+, lib
+, pulseaudio
+}:
+
+let
+  desktopItem = makeDesktopItem {
+    name = "qpaeq";
+    exec = "@out@/bin/qpaeq";
```

Comment:
Wait, how does `@out@` get replaced?

Reply:
Don't know. It's copied from anydesk package.